ceph.spec.in

```diff
@@ -123,8 +127,8 @@ BuildRequires:	keyutils-libs-devel
 BuildRequires:	libatomic_ops-devel
 Requires:	gdisk
 Requires(post):	chkconfig
-Requires(preun):chkconfig
-Requires(preun):initscripts
+Requires(preun):	chkconfig
+Requires(preun):	initscripts
```

Comment:
In the future, this is one of those whitespace changes I'd prefer to be broken out into a separate commit... but I'll let it slide for this PR :)